.github/workflows/ci_pr.yaml

```diff
@@ -0,0 +1,35 @@
+name: 'Lint PR title'
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+permissions:
+  pull-requests: read
+
+jobs:
+  validate-pr-title:
+    name: Validate PR title
+    runs-on: ubuntu-latest
+    steps:
+      - uses: amannn/action-semantic-pull-request@v5
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+            scopes: |
+              git
+              types
+              backend
+              client
```

Comment:
this scopes can be improved at any moment, this were the scopes we had originally when I created the first version that was not working.